frontend/packages/ceph-storage-plugin/src/components/namespace-store/create-namespace-store.tsx

```diff
@@ -22,8 +22,16 @@ const CreateNamespaceStore: React.FC<CreateNamespaceStoreProps> = ({ match }) =>
         <BreadCrumbs
           breadcrumbs={[
             {
-              name: 'Openshift Data Foundation',
-              path: resourcePathFromModel(ClusterServiceVersionModel, appName, ns),
+              name: appName ? 'Openshift Container Storage' : 'Openshift Data Foundation',
+              path: appName
+                ? resourcePathFromModel(ClusterServiceVersionModel, appName, ns)
+                : '/odf',
+            },
+            {
+              name: t('ceph-storage-plugin~Namespace Store'),
+              path: appName
```

Comment:
My suggestion will be to use `useFlag` and check ODF is installed or not.
Not sure about `appName`

Reply:
`appName` is truthy when OCS exists and ODF does not.

Reply:
`appName` is not reliable and looks hacking please use the Flag instead.